src/transformStateWithClones.js

```diff
@@ -7,7 +7,60 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    const { type } = action;
+
+    if (type === 'addProperties') {
```

Comment:
Describe all variants of cases in constant.
const ACTION_CASES = {
add: 'addProperties',
}

Reply:
Understand, thank you)